index.html

```diff
@@ -67,30 +74,55 @@ <h2>
     <h2>
       Model
     </h2>
-    <dl class="definitions" data-sort="" data-cite="vc-data-model">
+
+    <p class="issue" title="Definitions under discussion">
+The goal of the definitions in this section is to re-use or establish
+terminology that is common across a variety of digital credential formats and
+protocols.
+    </p>
+
+    <dl class="definitions" data-sort="" data-cite="vc-data-model-2.0">
       <dt>
         <dfn>Digital credential</dfn>
       </dt>
       <dd>
-        <p>
-          Is a [=verifiable credential=] about a person.
-        </p>
+        A cryptographically secured digital document containing one or more
+        [=claims=] made by an [=issuer=] about a [=subject=]. Examples include
+        ISO mobile driver's licenses [[ISO18013-5]] and W3C [=verifiable
+        credentials=] [[VC-DATA-MODEL-2.0]].
+      </dd>
+      <dt>
+        <dfn>Credential provider</dfn>
```

Comment:
Alternates include:

* Credential store (which is what CREDMAN calls this)
* Credential repository (which is what VCDM calls this)

Reply:
Yeah, we wanted something that covers basically these things being "remote" too (i.e., you can pull them from the browser, or multiple wallet apps, etc.)... so it's kinda like a manager?

Reply:
Yes, agreed that "manager" is probably the better term. CREDMAN uses it in a slightly different way (AND doesn't define it)... 

https://www.w3.org/TR/credential-management-1/#introduction

"Credential manager" in the CREDMAN usage of the term ties the "credential manager" to the user agent, often referring to it as "the user agent's credential manager". For this API, the two aren't tightly coupled, where the "credential manager" (digital wallet) is typically a completely separate application from the "user agent" (web browser).

Reply:
"Wallet" feels very concrete and product...y..., but it's definitely the right concept. 

However, I'm hesitant to use "wallet" because it's a loaded from a UI perspective. 

I guess what we'd like to convey here is more around the "role" that this bit of middleware plays... like, it may be a piece of software that acts as a wallet, but also allows you to select other wallets... plus coordinates the flow of data between itself and other wallets (as well as presenting UI).

Reply:
> I guess what we'd like to convey here is more around the "role" that this bit of middleware plays... like, it may be a piece of software that acts as a wallet, but also allows you to select other wallets... plus coordinates the flow of data between itself and other wallets (as well as presenting UI).

Sounds like a "mediator" to me, but "credential mediator" is the role the browser is playing, not the role the wallet is playing?

"Wallet" is not at the right conceptual level... a wallet is one form of this role, but it's not the only form.

I'll give "credential manager" a shot if we think that "credential provider" isn't going to work out.

Updated in 964c9c410943caf65e2d1e634b892d2d637c1afc.

Reply:
> I'll give "credential manager" a shot if we think that "credential provider" isn't going to work out.

"manager" is highly overloaded in Web Platform, and "credential manager", specifically means a lot of other things. "credential provider" isn't great, but I think is less ambiguous than "credential manager".

But also, I think we are missing one big architectural element here (speaking only for the choices that Android has made so far), which is that the [=user agent=] has a "registry" of all of the credentials from [=holders=] which it uses to build a cross-wallet credential selector. That is, I think we need to find a way to introduce this level of aggregation across wallets, because that's what the JS API "talks to", rather than directly to the actual [=holders=] up front.

Am I making sense?

Reply:
Yeah, that makes sense @samuelgoto. I think you articulated it well.

Reply:
>  "credential provider" isn't great, but I think is less ambiguous than "credential manager".

*lol* -- alright, back to "credential provider" it is! I'll make that change in an upcoming commit. Having a list of "managers" in the WebIDL felt off to me, but at least we tried it out and decided it's a direction we didn't want to go in.

> That is, I think we need to find a way to introduce this level of aggregation across wallets, because that's what the JS API "talks to", rather than directly to the actual [=holders=] up front. Am I making sense?

Yes, I think I get the general concern you're highlighting. Ignore the words I'm using for now, just trying to identify concepts... I believe you're identifying the "matching services" (the sandboxed WASM code), provided by each "credential provider" (digital wallet).

I was trying to avoid talking about those concepts in this PR (don't want the PR to add so much that it becomes a permathread and never gets merged because it's trying to do too much). Would it be okay if we introduced these concepts in a second PR? It seems orthogonal to the concept of a "credential provider"?

Reply:
(see suggestion about dropping "credential manager" entirely... that way we just side step this whole discussion, same as Cred Man does)

Reply:
Done in 9e8500ef18e3f42c349d7e0652f25d5fdb7d77b0.